index.html

```diff
@@ -44,9 +44,7 @@
         </p>
       </div>
 
-      <div class="connect">
```

Comment:
- [ ] I think it would be better if you use the `<section>` tag instead `<main>` tag
<img width="665" alt="Screen Shot 2021-11-24 at 12 51 29 PM" src="https://user-images.githubusercontent.com/69011963/143202412-714fa8e8-52d1-4e07-b2dc-94164aab0395.png">